src/lib/Preview.js

```diff
@@ -1299,7 +1309,7 @@ class Preview extends EventEmitter {
         if (canDownload(this.file, this.options)) {
             this.ui.showDownloadButton(this.download);
 
-            if (checkFeature(this.viewer, 'print')) {
+            if (this.canPrint()) {
```

Comment:
Probably move this outside the canDownload check to flatten the conditions?

Reply:
Sure